topics/metagenomics/tutorials/metagenomics-binning/turorial.md

```diff
@@ -0,0 +1,418 @@
+>    {{ page.zenodo_link }}/files/Assembly_with_MEGAHIT_on_ERR2231568.fasta
+>    {{ page.zenodo_link }}/files/Assembly_with_MEGAHIT_on_ERR2231569.fasta
+>    {{ page.zenodo_link }}/files/Assembly_with_MEGAHIT_on_ERR2231570.fasta
+>    {{ page.zenodo_link }}/files/Assembly_with_MEGAHIT_on_ERR2231571.fasta
+>    {{ page.zenodo_link }}/files/Assembly_with_MEGAHIT_on_ERR2231572.fasta
+>    ```
+>
+>    {% snippet faqs/galaxy/datasets_import_via_link.md %}
+>    {% snippet faqs/galaxy/datasets_import_from_data_library.md %}
+>
+>    > <comment-title></comment-title>
+>    > In case of large dataset, we can use FTP server or the [Galaxy Rule-based Uploader]({% link topics/galaxy-interface/tutorials/upload-rules/tutorial.md %}).
+>    {: .comment}
+>
+> 3. Create a paired collection named `Raw reads`, rename your pairs with the sample name
+>
+>    {% snippet faqs/galaxy/collections_build_list_paired.md %}
+>
+{: .hands_on}
+
+# Binning
+
+As explained before, there are many challenges to metagenomics binning. The most common of them are listed below:
+
+- High complexity.
+- Fragmented sequences.
+- Uneven coverage.
+- Incomplete or partial genomes.
+- Horizontal gene transfer.
+- Chimeric sequences.
+- Strain variation.
+
+![Image show the binning process where sequences are grouped together based on genome signatures like the kmer profiles of each contig, contig coverage, or GC content](./images/binning.png "Binning"){:width="60%"}
+
+In this tutorial we will learn how to use **MetaBAT2** tool through Galaxy:
+
+- **MetaBAT2**: Metagenome Binning based on Abundance and Tetranucleotide frequency
+
+  *Grouping large fragments assembled from shotgun metagenomic sequences to deconvolute complex microbial communities, or metagenome binning, enables the study of individual organisms and their interactions. Here we developed automated metagenome binning software, called MetaBAT, which integrates empirical probabilistic distances of genome abundance and tetranucleotide frequency. On synthetic datasets MetaBAT on average achieves 98percent precision and 90% recall at the strain level with 281 near complete unique genomes. Applying MetaBAT to a human gut microbiome data set we recovered 176 genome bins with 92% precision and 80% recall. Further analyses suggest MetaBAT is able to recover genome fragments missed in reference genomes up to 19%, while 53 genome bins are novel. In summary, we believe MetaBAT is a powerful tool to facilitate comprehensive understanding of complex microbial communities.*
+
+We will use the uploaded assembled fasta files as input to the algorithm (For simplicity reasons all other parameters will be preserved with their default values).
+
+> <hands-on-title>Individual assembly of short-reads with MEGAHIT</hands-on-title>
+> 1.  {% tool [ MetaBAT2](toolshed.g2.bx.psu.edu/repos/iuc/megahit/megahit/1.2.9+galaxy0) %} with parameters:
+>     - *"Fasta file containing contigs"*: `assembly fasta files`
```

Comment:
Is it not the `Raw reads` collection there?

Reply:
MetaBAT2 needs assembled reads as input, in order to identify the respective clusters. This is the reason I am using the assembled fasta from the metagenomics assembly tutorial